pages/linux/apt.md

```diff
@@ -0,0 +1,27 @@
+# apt
+
+> Debian and Ubuntu package management utility.
+
+- Synchronize list of packages and versions available. This should be run first, before running subsequent apt commands:
```

Comment:
> This should be run first, before running subsequent apt commands

While useful, I don't think its compulsory to run it every time, which is what this line implies. It might be better to say "Useful to run before searching or installing new packages".